students/vkis/lesson06/mailroom4.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python
+
+# Lession 06 - Mailroom v4
+# =================================================================
+
+
+# initialize list variables
+names = ['Tony Stark', 'Captain America', 'Daisy Johnson', 
+         'Melinda May', 'Phil Coulson']
+money = [906.04, 4500.00, 14.97, 555.02, 9999.99]
+gifts = [2, 2, 3, 2, 1]
+
+dict_data = {names[i]: [money[i], gifts[i]] for i in range(len(names))}
```

Comment:
while clever, I think it would have been more readable just to define it all in one dictionary

Reply:
also you can just do `for name in names:` no need for the `range(len(names))` thing